common/models.py

```diff
@@ -0,0 +1,107 @@
+import logging
+
+from django.db import models
+from django.utils import timezone
+from django.conf import settings
+
+LOGGER = logging.getLogger(__file__)
+
+CONTACT_TYPES = (
```

Comment:
Let contact types too be configurable via a simple model ( only a unique `name` field ).

Yes, it appears to be overkill at present; do it still.